modules/managers/galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/main/java/dev/galasa/zos3270/internal/comms/NetworkThread.java

```diff
@@ -217,11 +217,13 @@ public void processMessage(InputStream messageStream) throws IOException, Networ
             // This advances through the message buffer to skip the header bytes after DATA-TYPE.
             buffer.get(new byte[4]);
             
+            boolean isSSCPLUDisplay = false;
```

Comment:
This whole method is getting pretty big. Is there any way to split it up into su-functions to make it more readable ?

Reply:
Addressed in https://github.com/galasa-dev/galasa/pull/301/commits/465cb1069e44853227cb9821fc91c43e49bb66fc